compute/Makefile

```diff
@@ -22,15 +22,15 @@ sql_exporter.yml: $(jsonnet_files)
 		--output-file etc/$@ \
 		--tla-str collector_name=neon_collector \
 		--tla-str collector_file=neon_collector.yml \
-		--tla-str 'connection_string=postgresql://cloud_admin@127.0.0.1:5432/postgres?sslmode=disable&application_name=sql_exporter' \
+		--tla-str 'connection_string=postgresql://cloud_admin@127.0.0.1:5432/postgres?sslmode=disable&application_name=sql_exporter&pgaudit.log=none' \
```

Comment:
I wonder, should we do the same for compute_ctl connections? Especially activity monitor, it runs a bunch of queries pretty often. It probably should be enough to put this option here https://github.com/neondatabase/neon/blob/24d7c37e6ee7b730f983487351721f40922a9745/compute_tools/src/compute.rs#L362

Reply:
> And then the pseudocode that you wrote actually already exists at

Yeah, but I meant that we should swap `options, EXTRA_OPTIONS` if we want cplane values to take precedence

Reply:
See the implementation at https://github.com/neondatabase/neon/pull/12262.